frontends/web/src/api/account.ts

```diff
@@ -105,7 +115,7 @@ export type TAccountsTotalBalanceResponse = {
     totalBalance: TAccountsTotalBalance;
 } | {
     success: false;
-    errorCode?: 'ratesNotAvailable';
+    errorCode?: 'ratesNotAvailable' | 'syncInProgress';
```

Comment:
Sorry for flip-flopping, but I checked and saw that we don't enumerate the error codes in other response types and have `string`. Maybe that makes more sense, as it's hard to know exactly what all possible error codes might be - what if one is missed? Then the frontend could crash in the worst case.

So let's change back to `string`, here and in the other instances. Sorry again.

Reply:
done, thanks